### ISSUE TITLE
How to get non-localized English-only event names?

### ISSUE THREAD

**Issue by josephernest** [boppreh/keyboard#533]
With 

    import keyboard
    event = keyboard.read_event()
    print(event.name)

I get

    haut    # I would prefer UP (or UP ARROW)
    maj      # I would prefer SHIFT

**How to disable localization and get only English event names, even on a French Windows?**

**Comment by boppreh** [comment]
I agree with it, and I'm working on it on a separate branch (see for example https://github.com/boppreh/keyboard/commit/275b3b466cb71b904aa02c8911e242e5a4b180af ).

The challenge is getting all of these types of keys right:

- Named after the characters they type ('q', highly dependent on layout).
- Unique names ('backspace').
- Sided modifiers ('left shift').
- Name changes depending on numlock ('numpad 5' becomes 'numpad home').
- Where Windows disagrees with common sense (did you know that the numpad 'home' is considered the primary, and the 'home' above the arrows is the "extended" key? and that 'print screen' is extended, but 'alt+print screen' is not?).
- Media keys ('play/pause media', and the fact it has a virtual key code but no scan code).
- Localized names ('haut').
- Fake keys, like the events generated by extra mouse buttons.

There are some 4 different ways of getting key names in Windows, and each of them gives good results for some keys, but awful results for others.

Regarding localized keys, it's good to at least keep track of them, so that if the user creates a hotkey 'maj+haut' the library knows what to do with it.

**Comment by josephernest** [comment]
Thanks for your answer and congrats again for your lib @boppreh.

Would you know a quick hack to set English everywhere at least for `keyboard.read_event().name`?
